manifests/candlepin.pp

```diff
@@ -120,17 +113,26 @@
       owner  => 'tomcat',
       group  => $group,
       mode   => '0750',
-    } ~>
-    exec { 'import CA into Candlepin truststore':
-      command => "keytool -import -v -keystore ${amqp_truststore} -storepass ${keystore_password} -alias ${alias} -file ${ca_cert} -noprompt",
-      creates => $amqp_truststore,
-    } ~>
-    exec { 'import client certificate into Candlepin keystore':
-      # Stupid keytool doesn't allow you to import a keypair.  You can only import a cert.  Hence, we have to
-      # create the store as an PKCS12 and convert to JKS.  See http://stackoverflow.com/a/8224863
-      command => "openssl pkcs12 -export -name amqp-client -in ${client_cert} -inkey ${client_key} -out /tmp/keystore.p12 -passout file:${password_file} && keytool -importkeystore -destkeystore ${amqp_keystore} -srckeystore /tmp/keystore.p12 -srcstoretype pkcs12 -alias amqp-client -storepass ${keystore_password} -srcstorepass ${keystore_password} -noprompt && rm /tmp/keystore.p12",
-      unless  => "keytool -list -keystore ${amqp_keystore} -storepass ${keystore_password} -alias amqp-client",
-    } ~>
+    } ->
+    java_ks { 'candlepin:truststore':
```

Comment:
Note this does this to the `amqp` truststore, we need a similar declaration for the `keystore` (thats what https://github.com/theforeman/puppet-certs/pull/245/files#diff-0a129cc5a540f220caade2da4fc3229bR113 is trying to solve)